app/assets/stylesheets/hyrax/_accessibility.scss

```diff
@@ -0,0 +1,28 @@
+// This stylesheet is a place to apply styling changes for improving the
+// accessibility of Hyrax vanilla
+
+// Testing: WAVE evaluation tool: http://wave.webaim.org/
+// Chrome has a WAVE extension that you can toggle on/off for testing pages
+
+// Bootsrap upto v4.0 has poor styling for color-contrast that causes the WAVE
+// tool to throw contrast errors
+// Read more: https://getbootstrap.com/docs/4.0/getting-started/accessibility/#color-contrast
+// These changes meet WCAG AA and AAA compliance standard
+// * Darkens the text for active breadcrumb on every page. Meets WCAG AA and AAA
+// * Darkens the text for disabled pagination buttons on every page.
+.breadcrumb>.active,
+.pagination > .disabled > a {
+  color: #4c4c4c;
```

Comment:
Can this be set by using a bootstrap variable?

Reply:
@jcoyne That's not a bad idea. The variable would need to come before bootstrap me thinks for it to work. Similar to what @lfarrell has done here:

https://github.com/samvera/hyrax/blob/master/app/assets/stylesheets/_bootstrap-default-overrides.scss

Reply:
I think `$btn-link-disabled-color` is the variable you want to set.